WordPress/Tests/Security/EscapeOutputUnitTest.php

```diff
@@ -160,10 +163,34 @@ public function getErrorList( $testFile = '' ) {
 					655 => 1,
 					657 => 1,
 					663 => 1,
-					664 => 1,
+					// PHPCS 3.13.3 changed the tokenization of FQN exit/die it impacts directly how this test case
+					// behaves (see https://github.com/PHPCSStandards/PHP_CodeSniffer/issues/1201).
+					664 => version_compare( $phpcs_version, '3.13.3', '>=' ) ? 1 : 0,
```

Comment:
I seem to remember we dropped support for PHPCS < 3.13.4 ? In which case, this toggle should no longer be needed.